packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import MockApp from '../../../.storybook/components/MockApp';
+import { Subscriptions, SubscriptionsProps } from './index';
+
+function init() {
+  storiesOf('routes/Subscriptions', module)
```

Comment:
Here's a slightly more complex story, where I'm trying some variations in React state to exercise a few aspects of the subscription management page.